### PR TITLE
Levinska_TaxRateTest

### PR DESCRIPTION
**Pull request by sofiialevinska**
I am thinking about adding the same logic as in currency tests - reading names and values from admin Tax Rate page to <String, String> Map. 
Now test is also working with hardcoded original values.

### CODE REVIEW

File: src/main/java/com/opencart/pages/HeaderPage.java [hunk 1]
```diff
@@ -28,17 +28,40 @@
...
+    private WebElement productEcoTaxRate;
+
+    @FindBy (css = ".table.table-bordered tr:nth-child(2) .text-right:first-child strong")
+    private WebElement productEcoTaxName;
+
+    @FindBy (css = ".table.table-bordered tr:nth-child(1) .text-right:last-child")
+    private WebElement productPrice;
+
+    @FindBy (css = ".table.table-bordered tr:nth-child(4) .text-right:last-child")
+    private WebElement productTotalPrice;
```
Comment (adomanich) [inline src/main/java/com/opencart/pages/HeaderPage.java:50]: Why did you locate all these elements into HeaderPage? I think it relates to ProductContainer

**Review by adomanich — CHANGES_REQUESTED** [review]

File: src/main/java/com/opencart/pages/HeaderPage.java (hunk 1 above)
Comment (sofiialevinska) [reply src/main/java/com/opencart/pages/HeaderPage.java:50]: This is WebElements from shopping cart from header button "ShoppingCart" (see screenshot).
I thought ProductContainer is that field with photo, Title, Description, Buttons: WishList, Shopping Cart, etc.. (see screenshot). 
Is it better to put the mentioned WebElements to ProductContainer?

Screen Shot 2020-11-30 at 11 22 24

Screen Shot 2020-11-30 at 11 21 17

**Review by cheesyy183 — APPROVED** [review]

File: src/main/java/com/opencart/pages/HeaderPage.java (hunk 1 above)
Comment (adomanich) [reply src/main/java/com/opencart/pages/HeaderPage.java:50]: Oh, yeah, you are right, I thought it is related to product characteristics

Comment (adomanich) [reply src/main/java/com/opencart/pages/HeaderPage.java:50]: But before my approval, please resolve conflicts for safely merge

Comment (sofiialevinska) [reply src/main/java/com/opencart/pages/HeaderPage.java:50]: @adomanich resolved

**Review by adomanich — APPROVED** [review]

**Review by cheesyy183 — APPROVED** [review]

**Pull request merged**